### PR TITLE
Update badges in readme

### PR DESCRIPTION
**Pull request by kianmeng**
Some upkeeping of badges of this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # gRPC Elixir
 
+[![GitHub CI](https://github.com/elixir-grpc/grpc/actions/workflows/ci.yml/badge.svg)](https://github.com/elixir-grpc/grpc/actions/workflows/ci.yml)
 [![Hex.pm](https://img.shields.io/hexpm/v/grpc.svg)](https://hex.pm/packages/grpc)
-[![Travis Status](https://app.travis-ci.com/elixir-grpc/grpc.svg?branch=master)](https://app.travis-ci.com/elixir-grpc/grpc)
```
Comment (kianmeng) [inline README.md:4]: Should we retire the project from Travis CI, as it's still active, see https://app.travis-ci.com/github/elixir-grpc/grpc?

Comment (sleipnir) [reply README.md:4]: Excelent

**Comment by sleipnir** [comment]
Thank you @kianmeng

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**